common/page.go

```diff
@@ -860,7 +860,10 @@ func (p *Page) QueryAll(selector string) ([]api.ElementHandle, error) {
 func (p *Page) Reload(opts goja.Value) api.Response {
 	p.logger.Debugf("Page:Reload", "sid:%v", p.sessionID())
 
-	parsedOpts := NewPageReloadOptions(LifecycleEventLoad, p.defaultTimeout())
+	parsedOpts := NewPageReloadOptions(
+		LifecycleEventLoad,
+		time.Duration(p.timeoutSettings.navigationTimeout())*time.Second,
```

Comment:
Reminder: This needs to be updated once https://github.com/grafana/xk6-browser/pull/1035 has been merged in.

Reply:
Read for additional context: https://github.com/grafana/xk6-browser/pull/1035#pullrequestreview-1630425853 :)